package.json

```diff
@@ -4,7 +4,8 @@
   "description": "practice unit tests in javascript",
   "main": "index.js",
   "scripts": {
-    "test": "jest"
+    "test": "jest",
+    "test:coverage":"npm run test -- --coverage"
   },
   "dependencies": {
     "jest": "^29.7.0"
```

Comment:
jest shouldn't be in the dependencies section

Reply:
I changed what needed to be changed.